templates/include/yarp/ast.h.erb

```diff
@@ -54,7 +59,7 @@ typedef uint16_t yp_node_flags_t;
 // all nodes (the ones listed below). Others are specific to certain node types.
 static const yp_node_flags_t YP_NODE_FLAG_STATEMENT = 0x1;
 static const yp_node_flags_t YP_NODE_FLAG_NEWLINE = 0x2;
-static const yp_node_flags_t YP_NODE_FLAG_STATIC_LITERAL = 0x3;
+static const yp_node_flags_t YP_NODE_FLAG_STATIC_LITERAL = 0x4;
```

Comment:
Why this change?

Reply:
The flags are a bit field, so it's important that they're all powers of 2, otherwise they won't work correctly. 0x3 was a bug

Reply:
Can we make the flags being powers of 2 more explicit then, perhaps in the style of CRuby, eg `0x1`, `0x1 << 1`, `0x1 << 2`?